### PR TITLE
React Level-2 KanBan Board app

### PR DESCRIPTION
**Pull request by D-sourav-M044**
Fixes #217 

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [X] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
New react project named kanban board has been added.
## What was done
In the kanban board project there are five status box. At first task needs to be added with the input task box. Then the task will be added to the the backlog status box. After that the task can drag and drop to any other status box.
## How to test
After pulling successfully- "npm i" will install the required npm_modules. Then "npm start" will run the project.
- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: Kanban Board/src/store/context.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const ThemeStore = (props) => {
+  return (
+    <ThemeContext.Provider value={props.theme}>
+      {props.children}
+    </ThemeContext.Provider>
+  );
+};
+
+export { TastStore, TaskContext, ThemeContext, ThemeStore };
```
[the remark this answers is not in the document]
Comment (D-sourav-M044) [reply Kanban Board/src/store/context.js]: Yes. It is updated now. [0b9a59f](https://github.com/CraftsmenLtd/coe-batch-q322/commit/0b9a59fd7d5b03774093085590e80dc69e5ef71f)

File: Kanban Board/src/store/reducer.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export const Reducer = (issues, action) => {
```
[the remark this answers is not in the document]
Comment (D-sourav-M044) [reply Kanban Board/src/store/reducer.js]: updated the name. [0b9a59f](https://github.com/CraftsmenLtd/coe-batch-q322/commit/0b9a59fd7d5b03774093085590e80dc69e5ef71f)

File: Kanban Board/src/App.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ThemeProvider } from "@emotion/react";
+import { ThemeStore } from "./store/context";
+import Home from "./home";
+import { darkMode, lightMode } from "./theme/theme";
+
+function App() {
+  const [enableDarkMode, setenableDarkMode] = React.useState(false);
```
[the remark this answers is not in the document]
Comment (D-sourav-M044) [reply Kanban Board/src/App.js]: Changing theme is now controlled from the context file. [0b9a59f](https://github.com/CraftsmenLtd/coe-batch-q322/commit/0b9a59fd7d5b03774093085590e80dc69e5ef71f)

File: Kanban Board/src/App.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  return (
+    <ThemeProvider theme={enableDarkMode ? darkMode : lightMode}>
+      <ThemeStore
+        theme={{
+          enableDarkMode: enableDarkMode,
+          setenableDarkMode: setenableDarkMode,
+        }}
```
[the remark this answers is not in the document]
Comment (D-sourav-M044) [reply Kanban Board/src/App.js]: Updated it. [0b9a59f](https://github.com/CraftsmenLtd/coe-batch-q322/commit/0b9a59fd7d5b03774093085590e80dc69e5ef71f)

File: Kanban Board/src/home/home.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import Board from "../components/board";
+import { HomeDiv, BodyDiv } from "./homeStyled";
+
+const Home = () => {
+  return (
+    <HomeDiv>
+      <NavBar />
+      <DndProvider backend={HTML5Backend}>
+        <BodyDiv>
+          <TastStore>
```
[the remark this answers is not in the document]
Comment (D-sourav-M044) [reply Kanban Board/src/home/home.js]: Yes. Updated it. [0b9a59f](https://github.com/CraftsmenLtd/coe-batch-q322/commit/0b9a59fd7d5b03774093085590e80dc69e5ef71f)

File: Kanban Board/src/components/navBar/navStyled.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import styled from "@emotion/styled";
```
[the remark this answers is not in the document]
Comment (D-sourav-M044) [reply Kanban Board/src/components/navBar/navStyled.js]: updated the name. [0b9a59f](https://github.com/CraftsmenLtd/coe-batch-q322/commit/0b9a59fd7d5b03774093085590e80dc69e5ef71f)

**Review by tusher-A — APPROVED** [review]
LGTM

**Pull request merged**